src/Share.php

```diff
@@ -61,6 +61,24 @@ public static function linkedin(?string $url = null, ?string $message = '')
         return $link;
     }
 
+    /**
+     * Get whatsup share link
+     *
+     * @param mixed $url to be shared.
+     * @param mixed $message to append.
+     * @return string $link to be used in href.
+     * @since 1.0
+     */
+    public static function whatsup($url = null, $message = '')
```

Comment:
- Add type to string param. In this case nullable string. ?string
- The code doesnt seem to use Message. is it a copypaste mistake? If it is unused, just remove it.